docs/user/masternodes/understanding.rst

```diff
@@ -317,3 +317,33 @@ to be determined, although some pointers can be taken from the `roadmap
 It should be possible to run Dash masternodes on normal VPS servers
 until the block size reaches approximately 20 MB, after which custom
 hardware such as GPUs and eventually ASICs may be required.
+
+High-Performance Masternodes (HPMNs)
+--------------
+
+High-Performance Masternodes (HPMNs) are a subset of Masternodes that have 
+been created to host the Dash Platform. An HPMN is a lot like Masternodes 
+with three main differences:
+
+- The collateral to own an HPMN is 4000 Dash as opposed to 1000 Dash 
+for regular masternodes.
+- The recommended specs for HPMNs are higher than those for 
+regular masternodes.
+- HPMNs serve Platform along with Core, while regular masternodes 
+only serve Core.
+
+HPMNs will provide all the services for Dash Platform. Regular 
+masternodes will not perform any Dash Platform services. In Dash Core 
+quorums, HPMNs will act just like regular masternodes. In the DAO, they 
+will have 4 times more voting power due to their collateral being 4 times greater.
```

Comment:
I think it could be helpful to introduce HPMNs earlier on the page. I'm thinking adding a "Types of Masternodes" section just before the "DIP003 Masternode Changes" section would be good and shifting this part of your additions there (and incorporating the table idea from the other PR).